tests/core/parameters_date_indexing/aad_rg.yaml

```diff
@@ -0,0 +1,121 @@
+description: Age d'annulation de la décote
```

Comment:
As openfisca-core is an international repository, use [English as default language](https://openfisca.org/doc/contribute/language.html#language) for the filename and content?

Reply:
Yes a translation is needed